src/content/docs/dev/send-tokens/interchain-tokens/minter-operator-roles.mdx

```diff
@@ -0,0 +1,107 @@
+
+## Operator Role
+
+The Operator role controls who can manage the token manager's configuration and operations. This role has administrative powers over how tokens are handled during cross-chain transfers.
+
+Based on the [`IOperator`](https://github.com/axelarnetwork/interchain-token-service/blob/main/contracts/interfaces/IOperator.sol) interface and contract implementations, an Operator can:
+
+1. **Set flow limits**: Control the rate at which tokens can flow into or out of a chain.
+2. **Approve service**: Grant permission to the ITS to operate on behalf of the token manager.
+3. **Transfer operatorship**: Transfer the operator role to another address.
+4. **Propose operatorship change**: Propose a new operator (two-step transfer process).
+
+### Implementation Details
+
+In the ITS, operators are often set during token manager deployment:
+
+```solidity
+// From InterchainTokenFactory.sol
+
+bytes memory linkParams = '';
+if (operator != address(0)) {
+    linkParams = operator.toBytes();
+}
+```
+
+The `linkParams` is often used to specify the operator address for the token manager.
+
+## Role Transitions and Ownership
+
+Both roles feature a secure two-step transfer process:
+
+1. The current role holder proposes a new address (`proposeOperatorship` or `proposeMintership`).
+2. The proposed address must accept the role (`acceptOperatorship` or `acceptMintership`).
+
+This prevents accidental transfers to invalid addresses.
+
+## Practical Applications
+
+### Minter Role Usage
+
+1. **Token Supply Management**: The minter can control the total supply by minting or burning tokens.
+2. **Cross-Chain Transfers**: For mint/burn token managers, the minter creates tokens on the destination chain equivalent to what was burned on the source chain.
+3. **Token Migration**: The [`migrateInterchainToken`](https://github.com/axelarnetwork/interchain-token-service/blob/main/contracts/InterchainTokenService.sol) function in ITS shows how mintership is transferred to optimize token operations.
+
+### Operator Role Usage
+
+1. **Flow Control**: Operators can set limits on how many tokens can flow in or out of a chain in a given period.
+2. **Security Governance**: Operators can implement security measures to protect against attacks.
+3. **Service Authorization**: Operators authorize the ITS to interact with token managers.
+
+## Role Interaction in ITS
+
+- The ITS contract initially holds the minter role for native interchain tokens.
+- The token manager typically becomes the minter to handle cross-chain transfers.
+- Custom operators can be specified for token managers.
```

Comment:
When would the manager be the operator? I thougt the operator is used to interact with certain manager functionality?